### PR TITLE
Disassemble KMR 3 Battle

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

File: src/battle/area_kmr_part_3/area.c [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+extern Stage N(kmr_05);
+extern Stage N(kmr_06);
+extern Stage N(nok_01);
+extern Stage N(mim_01);
+extern Stage N(sam_01);
+extern Stage N(mac_01);
+extern Stage N(kpa_01);
+
+Formation N(formation_00) = {
+    { &N(spiked_goomba), .home = { .index = 2 }, .priority = 0 },
```
Comment (nanaian) [inline src/battle/area_kmr_part_3/area.c:28]: I thought in the other areas you changed this to look like the following?
```suggestion
    { .actor = &N(spiked_goomba), .home = { .index = 2 }, .priority = 0 },
```

**Comment by BowserSlug** [comment]
✅ This PR fixes 4 warnings!

File: src/battle/area_kmr_part_3/area.c (hunk 1 above)
Comment (JoshDuMan) [reply src/battle/area_kmr_part_3/area.c:28]: Maybe I did? I didn't know it was different from the script, I'll adjust it

Comment (nanaian) [reply src/battle/area_kmr_part_3/area.c:28]: As long as its consistent :)

Comment (Brotenko) [reply src/battle/area_kmr_part_3/area.c:28]: I'd personally like it to look like the suggested change tbh - In case there is still a decision to be made

Comment (nanaian) [reply src/battle/area_kmr_part_3/area.c:28]: Alright add `.actor =` to the tool and these formations and this PR is good for me

File: ver/us/splat.yaml [hunk 2]
```diff
@@ -3746,19 +3746,29 @@ segments:
...
+    - [0x448420, c, stage/kmr_02]
+    - [0x448420, c, stage/kmr_03]
+    - [0x448690, c, stage/kmr_04]
+    - [0x448900, c, stage/kmr_05]
+    - [0x448B70, c, stage/kmr_06]
+    - [0x448E20, c, stage/nok_01]
+    - [0x448E20, c, stage/mim_01]
+    - [0x448E80, c, stage/sam_01]
+    - [0x4491A0, c, stage/mac_01]
+    - [0x4495F0, c, stage/kpa_01]
```
Comment (ethteck) [inline ver/us/splat.yaml:3771]: Technically, this should end in `- [0x4495F0]` to be consistent. But I'm starting to think we should just remove these closing ones after a list of c segments across the board. for now let's add it back though just for consistency. unless you wanna remove all of them here

**Review by ethteck — CHANGES_REQUESTED** [review]

**Review by nanaian — APPROVED** [review]

**Pull request merged**